src/lib/services/blindbid/verify_score_handler.rs

```diff
@@ -39,15 +38,8 @@ where
             BlsScalar::from(self.request.get_ref().round as u64);
         let latest_consensus_step =
             BlsScalar::from(self.request.get_ref().step as u64);
-        // Get bid from storage
-        let (bid, branch) = get_bid_storage_fields(
-            self.request.get_ref().index_stored_bid as usize,
-            // XXX: These values are hardcoded so that we can mock the
-            // representation inside of the tree of the same Bid that should
-            // be previously allocated.
-            None,
-            None,
-        )?;
+        // Get bid from storage (FIXME: Once Bid contract is done and working.)
+        let (bid, branch) = unimplemented!();
```

Comment:
This is expected to be in the next version?

Reply:
We haven't started versioning. We're simply building rusk. So I'll not worry for now about it.